javascript/src/seedsource/components/RunStep.jsx

```diff
@@ -2,63 +2,97 @@ import React  from 'react'
 import PropTypes from 'prop-types'
 import ConfigurationStep from 'seedsource/containers/ConfigurationStep'
 import SaveModal from 'seedsource/containers/SaveModal'
+import ModalCard from 'seedsource/components/ModalCard'
 import Dropdown from 'seedsource/components/Dropdown'
 import { reports } from '../../config'
+import { Component } from 'react'
+import Map from 'seedsource/containers/Map'
+import ReportButton from 'seedsource/containers/ReportButton'
 
-const RunStep = props => {
-    let {
-        number, configuration, canRun, canSave, isLoggedIn, reportIsFetching, onRun, onSave, onExport, onExportTIF
-    } = props
+class RunStep extends Component {
+    constructor(props) {
+        super(props)
+        this.state = {
+            previewModal: false,
+            exportType: null
+        }
+    }
 
-    return (
-        <ConfigurationStep title="Map your Results" number={number} name="run" active={false}>
-            <div>
-                <h4></h4>
-                <button
-                    className="button is-primary is-large is-fullwidth"
-                    disabled={!canRun}
-                    onClick={e => {
-                        onRun(configuration)
-                    }}
-                >
-                    Run Tool
-                </button>
-            </div>
-            <div className="margin-top-10">
+    render() {
+        let {
+            number, configuration, canRun, canSave, isLoggedIn, reportIsFetching, onRun, onSave, onExport, onExportTIF
+        } = this.props
+        let button = <div onClick={() => this.setState({previewModal: false})} className="tabs is-toggle">
+                        <ul className="report-button">
+                            <ReportButton name={this.state.exportType}
+                                >Export
```

Comment:
Please clean up this formatting, and give the button the `is-primary` class.

Also, there are a lot of buttons in this component, please use a more descriptive variable name.